docker-compose.yml

```diff
@@ -11,19 +11,6 @@ services:
     ports:
       - '6379:6379'
 
-  hmpps-auth:
-    image: quay.io/hmpps/hmpps-auth:latest
-    networks:
-      - hmpps
-    container_name: hmpps-auth 
-    ports:
-      - "9090:8080"
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8080/auth/health"]
-    environment:
-      - SPRING_PROFILES_ACTIVE=dev
-      - DELIUS_ENABLED=false
-
```

Comment:
Do we want to remove auth from docker-compose.yml? I'm not sure anyone now runs the entire thing locally, as requires too many containers. However, maybe we still could? Or create two -dev and -dev-api.yaml files for different flavours of running locally?

Reply:
We dont have the required roles in auth I thought? So if we want to keep it for local we need to add the roles in